test/service/actions/preferences.test.js

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import assert from 'assert';
+
+import * as Actions from 'service/actions/preferences';
+import Client from 'service/client';
+import configureStore from 'app/store';
+import {RequestStatus} from 'service/constants';
+import TestHelper from 'test/test_helper';
+
+describe.only('Actions.Preferences', () => {
```

Comment:
you also need to remove the `only` here so every other test gets run, that's probably why it didn't fail the other tests

```
Actions.Preferences
    ✓ getMyPreferences (363ms)
    ✓ savePrefrences (334ms)
    ✓ deletePreferences (307ms)


  3 passing (1s)
```